tests/dummy/lib/template-stuff/index.js

```diff
@@ -11,10 +11,21 @@ module.exports = {
     return true;
   },
 
+  init(parent) {
+    this._super.init && this._super.init.apply(this, arguments);
+
+    let project = parent;
+    while (project.parent) {
+      project = project.parent;
+    }
+
+    this.isDisabled = project.name() !== 'ember-css-modules';
```

Comment:
[`project.name()`](https://ember-cli.com/api/classes/Project.html#method_name) is private API, but I don't really see a public API way to access the name of the project. 🤷‍♂️ 

Should I add a comment or is this fine?

Reply:
I think it's fine. Even though it's marked private, I'd guess usage is [widespread enough](https://emberobserver.com/code-search?codeQuery=project.name) to qualify at least as intimate API at this point